kvariants/build.rs

```diff
@@ -0,0 +1,38 @@
+use serde::Deserialize;
+use std::error::Error;
+
+#[derive(Deserialize)]
+pub struct TsvRow {
+    lhs: String,
+    relation: String,
+    rhs: String,
+}
+
+fn main() {
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    println!("cargo:rerun-if-changed=dictionaries/source/kVariants.tsv");
+
+    compress_kvariant_txt().unwrap();
+}
+
+fn compress_kvariant_txt() -> Result<(), Box<dyn Error>> {
+    let reader = csv::ReaderBuilder::new()
+        .delimiter(b'\t')
+        .has_headers(false)
+        .from_path("dictionaries/source/kVariants.tsv");
```

Comment:
We should have an absolute path.
```suggestion
    let manifest_dir = env::var_os("CARGO_MANIFEST_DIR").unwrap();
    let src_path = Path::from(manifest_dir)
        .join("dictionaries")
        .join("source")
        .join("kVariants.tsv");
    let reader = csv::ReaderBuilder::new()
        .delimiter(b'\t')
        .has_headers(false)
        .from_path(src_path);
```

Reply:
Thanks! I wasn't aware of `CARGO_MANIFEST_DIR`. Fixed in d5413b1.